knip.json

```diff
@@ -6,5 +6,12 @@
       "ignoreBinaries": ["gatsby", "packages/cli/bin.js"]
     }
   },
-  "ignore": ["__fixtures__/**"]
+  "jest": {
+    "config": ["package.json"],
+    "entry": [
+      "**/__tests__/**/*.ts",
+      "**/?(*.)+(test).ts",
+      "**/__fixtures__/**/*.[jt]s"
+    ]
+  }
```

Comment:
q: what do we need this for?

Reply:
For some reason knip was failing to [automaticaly detect our jest usage](https://knip.dev/features/script-parser), and hence marking most of the test files as unused files.